### DISCUSSION TITLE
[Feature Request] Private Mode

### DISCUSSION THREAD

**Discussion by Saroufim** [andyholmes/valent#690] [Feature Request]
Sometimes other people are looking at your screen and you might want to just pause the notification popup for an hour or two. It would be handy to have a DND or private mode button that you can turn on and off. It wouldn't disable other useful info like battery status.

**Comment by andyholmes** [comment]
I think this is something better left to the desktop environment, no? Why only mute remote notifications if you're concerned about privacy?

**Comment by Saroufim** [comment]
Let's say I'm showing a coworker how I do something and I have to share my screen (physically). I obviously would want email notifications to come in but I wouldn't want Tinder to let everyone at the office know I got a new match while I'm sharing my screen for example

**Reply by andyholmes** [reply]
I don't know, that seems like a pretty narrow use case. I'd be open to hearing a more common scenario, but this sounds like a fair amount of work with not a lot of pay off.
